testing/mariadb-connector-odbc/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Joe Searle <joe@jsearle.net>
+# Maintainer: Joe Searle <joe@jsearle.net>
+pkgname="mariadb-connector-odbc"
+pkgver="3.1.4"
+pkgrel=0
+pkgdesc="MariaDB Connector/ODBC is a standardized, LGPL licensed database driver using the industry standard ODBC API."
+url="https://github.com/MariaDB/mariadb-connector-odbc"
+arch="all"
+license="LGPL-2.1-only"
+depends="unixodbc"
+makedepends="cmake unixodbc unixodbc-dev mariadb-connector-c mariadb-connector-c-dev mariadb-static"
```

Comment:
Are really both mariadb-connector-c and mariadb-connector-c-dev required?

Reply:
`mariadb-connector-c-dev` is required but that package depends upon `mariadb-connector-c`. I have removed `mariadb-connector-c` as a make dependency as `mariadb-connector-c-dev` will pull it down as a dependency anyway.

I have also removed `unixodbc` as a make dependency as that isn't required.